### DISCUSSION TITLE
Spellchecking in search bar is unnecessary

### DISCUSSION THREAD

**Discussion by Alwaysin** [FreshRSS/FreshRSS#6938] [Ideas]
There is already `autocomplete="off"`, what do you think about adding `spellcheck="false"`?
Not that the red underline in Chrome is bothering me too much, but a little bit ^_^
Maybe there are other text areas where this could be applied.

Speaking about `autocomplete="off"`, why is it there in the first place? I sometimes want to search things I've searched before and that would come in handy (I don't want to put them in user queries).

**Comment by Frenzie** [comment]
Firefox will persist what's in there on reloads if you don't set autocomplete=off. Maybe that's the reason?

**Reply by math-GH** [reply]
The spellcheck is disabled in single line inputs in my Firefox.

@Alwaysin : So you have the same issue i.e. also in the sharing service settings? Which browser do you use?

**Reply by Alwaysin** [reply]
@math-GH yes, I also have the spellcheck in eg. "Share name to display" and "Share URL to use" in the settings. Also in "Username" when I try to log in.
I'm using Chrome 130 on Windows 11.

**Reply by math-GH** [reply]
In Firefox I can enable/disable the <input> spellcheck with a mouse right click

**Comment by Alkarex** [comment]
I do not believe we have any `autocomplete` attribute in the search bar, do we? I think the search bar should be left with default browser settings.

**Reply by math-GH** [reply]
yes, there is no autocomplete attribute

**Reply by Alwaysin** [reply]
Okay, you're right, when using private mode in Chrome there is no such attribute. In "normal mode", it gets added when clicking inside the search bar. I have the same behaviour when disabling every extensions, so maybe a configuration of Chrome or I need to restart the browser after disabling the extensions - I'll do other tests later.

**Reply by Alkarex** [reply]
Note that to find out what HTML is sent by FreshRSS, you should look at the source code of the page, not inspect the DOM, which might have been modified by many things. We do not discriminate by user agent or other parameters, so it is the exact same HTML that is sent to Chrome, Firefox, Edge, etc. and also in whichever special mode you may be using (incognito, private...).
